### ISSUE TITLE
Duplicate deletion requests (and also uploads?)

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#3894]
From Ciell on our google groups forum (with some paraphrasing from me):

> I noticed my app uploaded one of my picture twice, they were named 2&3, but I saw the new feature to nominate for deletion. But now (after nominating one file for deletion) both files are nominated for deletion. And the DR's (deletion requests) are incomplete (they even seem to be missing if I want to read what the automated text was).
> 
> So screenshot attached and I'll clean up the images if no one has tomorrow - I hold admin rights on Commons, so no stress. Just thought you should know.

![Screenshot_20200803-231721_Commons](https://user-images.githubusercontent.com/3611199/89428154-6d223c00-d77f-11ea-937f-a920d4c569c9.jpg)

**Comment by sivaraam** [comment]
If I understand the report correctly, both [File:Coffeeshop Sensi (Winterswijk) 2.jpg](https://commons.wikimedia.org/wiki/File:Coffeeshop_Sensi_(Winterswijk)_2.jpg) and [File:Coffeeshop Sensi (Winterswijk) 3.jpg](https://commons.wikimedia.org/wiki/File:Coffeeshop_Sensi_(Winterswijk)_3.jpg) were nominated for deletion. But I don't see any hints of deletion nomination in the former. Surprisingly, I don't see clues of _any_ deletion nomination regarding the corresponding file in [Ciell's talk page](https://commons.wikimedia.org/wiki/User_talk:Ciel) and its history.

I only see "File:Coffeeshop Sensi (Winterswijk) 3.jpg" listed in [Commons:Deletion_requests/Archive/2020/08/03](https://commons.wikimedia.org/wiki/Commons:Deletion_requests/Archive/2020/08/03#File:Coffeeshop_Sensi_(Winterswijk)_3.jpg).

So, I couldn't find the duplicate deletion request, at least. Correct me if I missed something.

**Comment by misaochan** [comment]
If I'm reading correctly, she cleaned the duplicates up as she has Commons admin rights (she said "tomorrow", but I only saw her email 2 days later).

**Comment by sivaraam** [comment]
Yeah, I understand that the duplicates have been cleaned. Its obvious from the fact that [File:Coffeeshop Sensi (Winterswijk) 3.jpg](https://commons.wikimedia.org/wiki/File:Coffeeshop_Sensi_(Winterswijk)_3.jpg) is now redirecting to [File:Coffeeshop Sensi (Winterswijk) 2.jpg](https://commons.wikimedia.org/wiki/File:Coffeeshop_Sensi_(Winterswijk)_2.jpg). What I'm trying to say is I don't see hints of a duplicate deletion nomination for the [image that has been retained](https://commons.wikimedia.org/wiki/File:Coffeeshop_Sensi_(Winterswijk)_2.jpg). So, I'm confused about this part of the report:

>  But now (after nominating one file for deletion) both files are nominated for deletion

How was it mentioned that both the files had been nominated for deletion? Did they see that both the images in Commons's deletion nomination page (indicative of a bug in deletion nomination logic)? Or did they see the "This image has been nominated for deletion" text for both the images in the app (indicative of a bug is nomination detection logic)?

I'm guessing it's the latter as I really couldn't find any traces for the former.
